### DISCUSSION TITLE
Spotify can not play some lists, search function does not work for spotify anymore

### DISCUSSION THREAD

**Discussion by SirY4** [MiczFlor/RPi-Jukebox-RFID#1786] [Q&A]
Hi Phonieboxers,

on March 8 Spotify seen to have some major issues (see Twitter and other sources).  About that time I started to have some issues with my modipy: 
1.Some spotify playlists and links stopped working (webinterface and RFID triggered)
2. when I search things in the iris Modiy webinterface, I no longer get Spotify listings but only local options. 
I am not sure if the two are connected.

I get the message : (syntax error and could not load search result) "unexpected end of JSON input" within the IRIS web interface.

I upgraded with the one-line-installer but it did not change anything.

Do you have any ideas what to do?

Thank you!

**Comment by DavidOrtmann** [comment]
I have exactly the same problem without changing anything from the setup, it previously worked like a charm. Anyone know anything?

**Reply by SirY4** [reply]
I lateron opened the issue #1801 , after reinstall. Does that help?

**Reply by DavidOrtmann** [reply]
Huh ... I didn't reinstall or change the authentication. If the problem persists, I will reinstall, but for now I just ignore the Iris interface.

**Reply by SirY4** [reply]
The problem was my only reason for a reinstall.
